R/get.R

```diff
@@ -87,6 +87,21 @@ get <- function(value = NULL,
   # merge the specified configuration with the default configuration
   active_config <- merge_lists(default_config, do_get(config))
 
+  active_config <-
```

Comment:
Then here you'd just do something like:

```R
is_expr <- vapply(active_config, is.expression, logical(1))
active_config[is_expr] <- lapply(active_config[is_expr], eval, envir = baseenv())
```

You'd need to think a little about the correct evaluation environment — I chose `baseenv()` here so that the code doesn't see attached packages, which I _think_ makes sense.

Reply:
This worked first time, and agree that using `baseenv()` makes sense.  My only additional change was to use `if(any(is_expr) ...` for the (slight) performance gain when no strings need to be evaluated.

Reply:
I *think* we need a recursive version of this.  The lists are not guaranteed to be one layer deep.

Reply:
@hadley , I've written a recursive version of the evaluation code (because the yaml header could be nested).  It would be great to have another review, please.